CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 ### Tobac Changelog
 
+_**Version 1.3.2:**_
+
+**Bug fixes**
+
+- Fixed a bug with Feature Detection that caused it to fail when using `weighted_abs` position [#148](https://github.com/tobac-project/tobac/pull/148)
+- Fixed a bug where adaptive_search within `linking_trackpy` was not working correctly [#140](https://github.com/tobac-project/tobac/pull/140)
+
+**Repository enhancements**
+
+- Added automatic code coverage reports [#124](https://github.com/tobac-project/tobac/pull/124)
+- Added automatic building of readthedocs documentation on pull requests
+
+
```

Comment:
Note that these changes from release 1.3.2 have not been merged back yet

Reply:
Yes, agreed. We should merge 1.3.2 back into the RC branches. I think we should be able to do this without a PR, are you happy if I just make a merge commit?

Reply:
Yes please go ahead :) Thanks!